recipes/jupyterhub/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.6.1" %}
+
+package:
+  name: jupyterhub
+  version: {{ version }}
+
+source:
+  fn: jupyterhub-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/ac/9b/ef3435cf8951d7463f4175a0066a8454e224a598b2159f521cc011f8c2b6/jupyterhub-{{ version }}.tar.gz
+  md5: 6bd795a82bf15a037d097991110b8713
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: pip install --no-deps .
```

Comment:
> Sure we need agreement on important decisions. However, I don't think this is major...

I think we have different operating definitions of "major". To me, installation is the most important part. The rest is basically metadata. 😉 

> ...and we already merged feedstocks with that syntax. So I think this ship has sailed.

Perhaps it comes across as I'm being too much of a stickler here. Maybe if it were just that syntax, I might not balk so much. Though we have pip installs that [don't use `--no-deps`]( https://github.com/conda-forge/vcversioner-feedstock/blob/d04fb486630944675739278056f64fce3edbc434/recipe/meta.yaml#L14 ) and here's [another]( https://github.com/conda-forge/jsonschema-feedstock/blob/master/recipe/meta.yaml#L14 ). This [one]( https://github.com/conda-forge/ipython-feedstock/blob/master/recipe/meta.yaml#L13 ) matches 4, which can have some [surprising effects]( https://github.com/conda-forge/staged-recipes/issues/528#issuecomment-216828570 ). We really need a standard like yesterday and we need to stick with it. Also, we can't be allowing things in because we did it before as there are many wrong things we did before that are out in the wild.

Sorry if this comes off as strong, it is merely a reflection of my concern for the current situation with these `script` lines.